docs/model_control_mode.md

```diff
@@ -0,0 +1,68 @@
+# Model Control Mode
+
+TorchServe now supports model control mode with two settings "none"(default) and "enabled"
+
+## Two ways to set Model Control
+1. Add `--model-api-enabled` to command line when running TorchServe to switch from none to enabled mode. Command line cannot be used to set mode to none, can only be used to set to enabled
+2. Add `model_control_mode=enabled` or `model_control_mode=none` to config.properties file
+
+Priority between cmd and config file follows the following [TorchServer standard](https://github.com/pytorch/serve/blob/c74a29e8144bc12b84196775076b0e8cf3c5a6fc/docs/configuration.md#advanced-configuration)
+* Example 1:
+  * Config file: `model_control_mode=none`
+
+    cmd line: `torchserve --start --ncs --model-store model_store --model-api-enabled`
+
+    Result: Model api mode enabled
```

Comment:
Describe here the values that were set in the config file and command line and which took precedence to make it clear to the reader.